graph/services/user/user.service.go

```diff
@@ -1,21 +1,19 @@
 package services
 
 import (
-	"fmt"
-
 	"github.com/PrameshKarki/event-management-golang/configs"
 	"github.com/PrameshKarki/event-management-golang/graph/model"
 	"github.com/doug-martin/goqu/v9"
+	"github.com/sirupsen/logrus"
 )
 
 func FindOneByEmail(email string) *model.User {
 	db := configs.GetDatabaseConnection()
-	ds := configs.GetDialect().Select("id", "email", "phone_number").From(configs.TABLE_NAME["USER"]).Where(goqu.Ex{"email": email})
+	ds := configs.GetDialect().Select("id", "email", "phone_number", "password").From(configs.TABLE_NAME["USER"]).Where(goqu.Ex{"email": email})
 	sql, _, _ := ds.ToSQL()
-	fmt.Println("SQL", sql)
+	logrus.Info("SQL", sql)
 	row := db.QueryRow(sql)
 	var user model.User
-	row.Scan(&user.ID, &user.Email, &user.PhoneNumber)
-	fmt.Println(user, "USER")
+	row.Scan(&user.ID, &user.Email, &user.PhoneNumber, &user.Password)
```

Comment:
assuming you've got correct db tags in model.User
```suggestion
	row.ScanStruct(&user) 
```

Reply:
Is it works without adding any kind of ORM?